README.md

```diff
@@ -78,6 +78,51 @@ createAction('ADD_TODO')('Use Redux');
 
 `metaCreator` is an optional function that creates metadata for the payload. It receives the same arguments as the payload creator, but its result becomes the meta field of the resulting action. If `metaCreator` is undefined or not a function, the meta field is omitted.
 
+##### `createActions(?actionsMap, ?...actionTypes)`
+
+Returns an object mapping action types to action creators. The keys of this object are camel-cased from the keys in `actionsMap`; the values are the action creators.
+
+`actionsMap` is an optional object with action types as keys, and whose values can be
+
+- a function, which is the payload creator for that action
+- an object with `payload` and `meta` keys, mapping to payload and meta creators
+    - `meta` is **required** in this case to distinguish it meaningfully from the function form above
+
+`actionTypes` is an optional list of positional arguments that are action type strings; these action types will use the identity payload creator.
+
+```js
+const { actionOne, actionTwo, actionThree } = createActions({
+  ACTION_ONE(key, value) {
+    return { [key]: value };
+  },
+
+  ACTION_TWO: {
+    payload(first, second) {
+      return [ first, second ];
+    }
```

Comment:
I'd suggest to use an arrow function, so they will become one liners.

```js
payload: (first, second) => [first, second],
meta: (first, second) => ({ first , second })
```

Also no spaces at the brackets please ;)